### PR TITLE
Fixes character setup not properly displaying preferences for moths and lizards

### PR DESCRIPTION
**Pull request by Tsar-Salat**
## About The Pull Request
Added a single line of code that applies AFTER icon_update reset, rather than before, which fixes issues where moth wings and lizard frills would disappear randomly

I also capitalized a word for consistency

## Why It's Good For The Game
Fixes are good

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl: RKz
fix: fixed lizard frills and moth wings disappearing after choosing a new one
code: capitalized a wing name
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
Before changes (Live Build)
https://user-images.githubusercontent.com/62388554/208824963-b0a0272c-e2aa-455a-8386-c887e320f5b6.mp4

After (Local Build)
https://user-images.githubusercontent.com/62388554/208825152-6a66d247-0e84-424b-9697-2a0b9fd2eee7.mp4

File: code/modules/client/preferences2/character_save.dm [hunk 1]
```diff
@@ -471,6 +471,8 @@
 
 	character.dna.features = features.Copy()
 	character.set_species(chosen_species, icon_update = FALSE, pref_load = TRUE)
+	character.dna.features = features.Copy()
```
Comment (PowerfulBacon) [inline code/modules/client/preferences2/character_save.dm]: This probably shouldnt be done twice. If set_species modifies DNA, do it after. If not then do it before

File: code/modules/surgery/organs/wings.dm [hunk 2]
```diff
@@ -120,7 +120,7 @@
 	icon_state = "mothwings"
 	flight_level = WINGS_FLIGHTLESS
 	basewings = "moth_wings"
-	wing_type = "plain"
+	wing_type = "Plain"
```
Comment (PowerfulBacon) [inline code/modules/surgery/organs/wings.dm:123]: unexplained change

**Review by PowerfulBacon — DISMISSED** [review]

File: code/modules/surgery/organs/wings.dm (hunk 2 above)
Comment (Tsar-Salat) [reply code/modules/surgery/organs/wings.dm:123]: Capitalized for consistency

File: code/modules/client/preferences2/character_save.dm (hunk 1 above)
Comment (Tsar-Salat) [reply code/modules/client/preferences2/character_save.dm]: Like this:
`if(character.set_species && ???)'
'....................character.dna.features = features.Copy()'
'....................character.set_species(chosen_species, icon_update = FALSE, pref_load = TRUE)'
'else'
'....................character.set_species(chosen_species, icon_update = FALSE, pref_load = TRUE)'
'....................character.dna.features = features.Copy()'


How do I set the if statement to tell if character.set_species has been modified?

Comment (Tsar-Salat) [reply code/modules/client/preferences2/character_save.dm]: nevermind, turns out you only need the `character.dna.features = features.Copy()` after to work

**Review by itsmeow — APPROVED** [review]

**Comment by DatBoiTim** [comment]
You broke IPCs preferences.

**Pull request merged**